src/components/Profile/RocketsProfile.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+
+const RocketsProfile = () => {
+  const rockets = useSelector((state) => state.rocketsReducer);
+  const reservedRockets = rockets.filter((rocket) => rocket.reserved);
+  return (
+    <div className="reserve-container">
+      <h1 className="profile-rockets-title">My Rockets</h1>
```

Comment:
- [ ] You have two `<h1>` tags on the same page, which is not a good practice in your markup, remember that this tag is reserved for the main title of each page because search engines use them to index the page, so please consider following this standard! 🤓 🔧

Reply:
Thank you @anagudelogu ,This is resolved.